python_modules/dagster/dagster_tests/daemon_sensor_tests/test_run_status_sensors.py

```diff
@@ -1239,58 +1240,59 @@ def test_logging_run_status_sensor(
     workspace_context: WorkspaceProcessContext,
     external_repo: ExternalRepository,
 ):
-    freeze_datetime = pendulum.now()
-    with pendulum.test(freeze_datetime):
-        success_sensor = external_repo.get_external_sensor("logging_status_sensor")
-        instance.start_sensor(success_sensor)
+    with environ({"OVERRIDE_ALEMBIC_QUIET": "False"}):
```

Comment:
Maybe a comment about why we're setting this here? Although I still think we should just never quiet - seems like it'd be very hard to diagnose a similar flaky test in the future to know this is the right fix.